molecule/default/prepare.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Import upgrade playbook
+  import_playbook: upgrade.yml
+
+- name: Import python playbook
+  import_playbook: python.yml
```

Comment:
Nitpicky, but I'm not sure the playbook name should stay the same when all that is being installed is `pip`.

Reply:
Hmmm...I agree that it is a little misleading, but I think it is less confusing if we keep all the repos consistent here.  What say you, @dav3r?  I will defer to the groupthink.

Reply:
I think I'm with @mcdonnnj on this one.  Otherwise, I can see us coming back to this later and saying "Why on earth is this thing called `python.yml`??"

Reply:
I _think_ any upstream changes to `python.yml` will still trickle down to `pip.yml` after renaming.  Is anyone certain of that, one way or another?

Reply:
Not sure- only one way to find out!

Reply:
See commit 7d308b5.

Reply:
🎸